build/js/live-editor.ui.js

```diff
@@ -973,15 +948,25 @@ window.LiveEditor = Backbone.View.extend({
         }
 
         this.bind();
+
+        // jquery.ui.slider.js is require for the toolbar and audio to work
+        
+        // don't set up the toolbar if we don't need to
+        // this.setupToolbar();
+
+        // this should only be loaded at startup if we know that the program
+        // being loaded has a sound in it
+        // if not, we should be able to load this on demand when a person
+        // types "getSound"
         this.setupAudio();
     },
 
     render: function() {
-        this.$el.html(Handlebars.templates["live-editor"]({
-            execFile: this.execFile,
-            imagesDir: this.imagesDir,
-            colors: this.colors
-        }));
+        //this.$el.html(Handlebars.templates["live-editor"]({
+        //    execFile: this.execFile,
+        //    imagesDir: this.imagesDir,
+        //    colors: this.colors
+        //}));
```

Comment:
don't render anymore because the HTML for this template is hard coded in index.html.  this was done so that we could start loading output.html sooner.